utils_cv/detection/keypoint.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import Dict, List
+import numpy as np
+import torch
+
+
+class COCOKeypoints:
+    """ Keypoints class for COCO datasets.
+
+    The interpretations of different categories of COCO Keypoints are the
+    same.  The only difference is the labels and skeleton (connection order)
+    for various categories.
+    """
+
+    def __init__(
+        self,
+        keypoints: np.ndarray,
+        meta: Dict[str, List] = None,
+    ):
+        """
+        Args:
+            keypoints:
+            meta:
+        """
+        if meta is not None:
+            self._meta = meta
+
+        self.keypoints = keypoints
+        if self.keypoints is not None:
+            self._init_keypoints()
+
+    def _init_keypoints(self):
+        self.keypoints = np.asarray(self.keypoints, dtype=np.float)
+        if self.keypoints.ndim != 3:
+            self.keypoints = self.keypoints.reshape(
+                (-1, len(self.meta["labels"]), 3)
+            )
+        assert self.is_valid()
+        self.standardize()
+
+    @property
+    def meta(self):
+        """ Define how to connect keypoints. """
+        return self._meta
+
+    @meta.setter
+    def meta(self, value):
+        self._meta = value
+
+    def is_valid(self) -> bool:
+        # shape must be (N, len(self.meta["labels"]), 3)
+        if self.keypoints.shape[1:] != (len(self.meta["labels"]), 3):
+            return False
+
+        # indexes in skeleton should be valid
+        if np.max(np.array(self.meta["skeleton"])) >= len(self.meta["labels"]):
+            return False
+
+        return True
+
+    def standardize(self) -> None:
+        """ Make sure invisible points' x, y = 0. """
+        self.keypoints[self.keypoints[..., 2] == 0] = 0
+
+    def as_tensor(self):
+        return torch.as_tensor(self.keypoints, dtype=torch.float32)
+
+    def get_lines(self) -> List[List[float]]:
```

Comment:
Can you please remove functions (if any) which are not used in this (or in future) PRs, e.g. get_lines() or any other functions.

Reply:
`get_lines()` is used by plot.  Other unused functions are removed.